packages/phone-number-privacy/test/salt-generation/query-quota.test.ts

```diff
@@ -1,62 +1,62 @@
 import { getPerformedQueryCount } from '../../src/database/wrappers/account'
-import QueryQuota from '../../src/salt-generation/query-quota'
-
-const ACCOUNT = '0x78dc5D2D739606d31509C31d654056A45185ECb6'
-const PHONE_NUMBER = '+1234567890'
-const queryQuota: QueryQuota = new QueryQuota()
+import { getRemainingQueryCount } from '../../src/salt-generation/query-quota'
+import { getContractKit } from '../../src/web3/contracts'
+import {
+  ContractRetrieval,
+  createMockAttestation,
+  createMockContractKit,
+  createMockWeb3,
+} from '../utils'
+import { mockAccount, mockPhoneNumber } from '../values'
 
+jest.mock('../../src/web3/contracts')
+const mockGetContractKit = getContractKit as jest.Mock
 jest.mock('../../src/database/wrappers/account')
 const mockPerformedQueryCount = getPerformedQueryCount as jest.Mock
 
-jest.mock('@celo/contractkit', () => {
-  const utils = require('../utils')
-  const attestation = utils.createMockAttestation(3, 3)
-  const kit = utils.createMockContractKit({
-    [utils.ContractRetrieval.getAttestations]: attestation,
-  })
-
-  return {
-    ...jest.requireActual('@celo/contractkit'),
-    newKit: () => kit,
-  }
-})
-
-describe('get remaining query count', () => {
-  beforeEach(() => {
-    jest.clearAllMocks()
-  })
-  it('calculates remaining query count', async () => {
+describe(getRemainingQueryCount, () => {
+  it('Calculates remaining query count for verified account', async () => {
+    const contractKitVerifiedNoTx = createMockContractKit(
```

Comment:
By mocking contractKit instead of spying on and mocking the internal methods of the former Class, we can exercise the code flows more thoroughly and accurately.